### PR TITLE
Added localization support

### LINKED ISSUE #2: Remove hard-coded English, i.e. make localisable
At the moment the wording used in the property editor is hard-coded English. Ideally this should be changed to use language files so that translations _could_ be added.

### PR DESCRIPTION
**Pull request by leekelleher**
Fixes #2

Changed the VS project type to be a Razor Class Library, (so the XML language files are embedded).

Added example localisations for English, along with a Google-translate of Danish, (please let a Danish-speaking person correct any of these).

### CODE REVIEW

**Comment by LottePitcher** [comment]
Thanks @leekelleher for having a look at this (and the other PRs). I look forward to reviewing them tomorrow.

PS didn't realise you spoke Danish... ;-)

**Comment by leekelleher** [comment]
@LottePitcher Ja, dansk jeg taler godt. Kamelåså!

File: src/AdminOnlyProperty/wwwroot/App_Plugins/AdminOnlyProperty/lang/da.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<language alias="da" intName="Danish" localName="Dansk" lcid="" culture="da-DK">
+    <area alias="adminOnlyProperty">
+        <key alias="packageName">Admin Kun Ejendom</key>
+        <key alias="labelDataType">Datatype</key>
+        <key alias="descriptionDataType">Datatypen, der skal ombrydes.</key>
```
Comment (greystate) [inline src/AdminOnlyProperty/wwwroot/App_Plugins/AdminOnlyProperty/lang/da.xml:6]:
```suggestion
<key alias="descriptionDataType">Datatypen der skal skjules.</key>
```

File: src/AdminOnlyProperty/wwwroot/App_Plugins/AdminOnlyProperty/lang/da.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<language alias="da" intName="Danish" localName="Dansk" lcid="" culture="da-DK">
+    <area alias="adminOnlyProperty">
+        <key alias="packageName">Admin Kun Ejendom</key>
```
Comment (greystate) [inline src/AdminOnlyProperty/wwwroot/App_Plugins/AdminOnlyProperty/lang/da.xml:4]:
```suggestion
        <key alias="packageName">Admin Only Property</key>
```

Don't really think the package name should be translated...

Comment (leekelleher) [reply src/AdminOnlyProperty/wwwroot/App_Plugins/AdminOnlyProperty/lang/da.xml:4]: @greystate Awww 😿

**Review by greystate — COMMENTED** [review]
As per @leekelleher's request, I - a Danish person - have looked over the _less than perfectly auto-translated_ keys :)

File: src/AdminOnlyProperty/wwwroot/App_Plugins/AdminOnlyProperty/lang/da.xml (hunk 2 above)
Comment (leekelleher) [reply src/AdminOnlyProperty/wwwroot/App_Plugins/AdminOnlyProperty/lang/da.xml]: @greystate I chatted with @LottePitcher, she says that you are correct. 👍

**Comment by LottePitcher** [comment]
Thanks for your review @greystate! Seems sensible to me to not translate the package name 👍

**Pull request merged**